app/assets/stylesheets/app.scss

```diff
@@ -11,7 +11,20 @@ body > div {
   padding: 60px 15px 0;
 }
 
-td nav { text-align: right;}
+table {
+  clear: both;
+
+  td, th { text-align: center; }
```

Comment:
@LucasAnderson1984 did you see these issues?

I would recommend running through `rubocop`, `scss-lint`, and `jshint`.